badger/schema.py

```diff
@@ -203,6 +203,8 @@ def validate_scalar(self, chunk):
         'yaxis': Str() | Seq(Str()),
         Optional('parameters'): MapPattern(Str(), PlotMode()),
         Optional('xaxis'): Str(),
+        Optional('xlim'): Seq(Scalar()),
+        Optional('ylim'): Seq(Scalar()),
```

Comment:
Use `FixedSeq(Scalar(), Scalar())` here and this is fine.

Reply:
fixed.